packages/contact-center/station-login/src/station-login/index.tsx

```diff
@@ -14,16 +16,27 @@ const StationLogin: React.FunctionComponent<StationLoginProps> = observer(({onLo
   const props = {
     ...result,
     teams,
-    loginOptions
+    loginOptions,
+    userRole: 'Agent',
```

Comment:
**I have added this to show, where we need to add uuip-wc-user-profile web component props.**